src/drag-and-drop/components/drop-zone.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { IDropZone } from "./types";
+import css from "./drop-zone-wrapper.scss";
+
+export interface IProps {
+  target: IDropZone;
+  highlight?: boolean;
+}
+
+// Just an image for now, but it can be extended in the future to include some text and other elements.
+export const DropZone: React.FC<IProps> = ({ target, highlight }) => {
+  const style={width: target.targetWidth, height: target.targetHeight};
+  return (
+    <div className={highlight? css.highlight: ""} style={style}>
+      {target.imageUrl && <img src={target.imageUrl} alt="drag target" width={target.targetWidth} height={target.targetHeight}/>}
+      <div>{target.targetLabel}
+      </div>
```

Comment:
This reads a little weird I would write the div on one line:
<div>{target.targetLabel}</div>